components/monitoring/logging/base/configure-logging/configure-logforwarder.yaml

```diff
@@ -14,10 +14,26 @@ spec:
       secret:
         name: cloudwatch-forwarder
       type: cloudwatch
+    - name: splunk
+      type: fluentdForward
+      url: https://http-inputs-rhcorporate.splunkcloud.com
+      secret:
+        name: splunk-forwarder        
+      groupBy: namespaceName
```

Comment:
I'm not finding documentation for groupBy being supported by anything other then cloudwatch

Reply:
True. Removed.